ansible/configs/kni-osp/default_vars.yml

```diff
@@ -0,0 +1,699 @@
+################################################################################
+################################################################################
+### Environment Structure
+################################################################################
+################################################################################
+
+#######################
+# OpenStack auth vars #
+# #####################
+
+osp_auth_username: FROM_SECRETS
+osp_auth_username_member: FROM_SECRETS
+osp_auth_password: FROM_SECRETS
+osp_auth_password_member: FROM_SECRETS
+osp_project_name: FROM_SECRETS
+osp_project_id: FROM_SECRETS
+
+osp_auth_url: FROM_SECRETS
+osp_auth_user_domain: FROM_SECRETS
+osp_auth_project_domain: FROM_SECRETS
+
+############
+# DNS vars #
+############
+
+osp_cluster_dns_server: FROM_SECRETS
+osp_cluster_dns_zone: FROM_SECRETS
+ddns_key_name: FROM_SECRETS
+ddns_key_secret: FROM_SECRETS
+
+
+################
+### Env vars ###
+################
+
+guid: CHANGEME                            # Your Global UNIQUE Identifier
+
+env_type: kni-osp          # Name of config to deploy
+cloud_provider: osp                     # Which AgnosticD Cloud Provider to use
+
+output_dir: /tmp/output_dir                # Writable working scratch directory
+email: name@example.com                   # User info for notifications
+
+ansible_user: cloud-user
+remote_user: cloud-user
+
+
+repo_method: satellite
+
+set_repositories_satellite_hostname: FROM_SECRETS
+set_repositories_satellite_activationkey: FROM_SECRETS
+set_repositories_satellite_org: FROM_SECRETS
+
+install_student_user: true
+student_password: Redhat01
+
+create_unused_security_groups: False
+
+############################
+## Openshift install vars ##
+############################
+
+openshift_fip_provision: true
+
+my_ocp_domain: "{{ osp_cluster_dns_zone }}"
```

Comment:
I think it would be good to change these variable names to something a little more appropriate.

Maybe `kni_xxxxx` instead of starting with `my`. Should be an easy find/replace and the var naming will be better.